wasm/Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [package]
 name = "iroha_wasm"
-version = "0.1.0"
-authors = ["Iroha 2 team <https://github.com/orgs/soramitsu/teams/iroha2>"]
+
 edition = "2021"
+version = "2.0.0-pre-rc.13"
+authors = ["Iroha 2 team <https://github.com/orgs/soramitsu/teams/iroha2>"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+license = "Apache-2.0"
```

Comment:
I'm not sure whether to use `license-file = ../LICENSE` here instead because this crate doesn't include the license and is not part of the workspace

Reply:
I prefer licence files. This is standard such as README.

Reply:
are you saying to keep this `license = "Apache-2.0"` and duplicate `LICENSE` file in the `wasm` crate. I think that's the best

Reply:
the choice here is:
1. use `license-file = ../LICENSE` in every workspace except root
2. use `license = "Apache-2.0"` and duplicate `LICENSE` file in every workspace we have (root, wasm, permission_validators)